src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -146,15 +167,12 @@ public ProjectContext CreateRuntimeContext(IEnumerable<string> runtimeIdentifier
                 return this;
             }
 
-            var standalone = !RootProject.Dependencies
-                .Any(d => d.Type.Equals(LibraryDependencyType.Platform));
-
             var context = CreateBuilder(ProjectFile.ProjectFilePath, TargetFramework)
-                .WithRuntimeIdentifiers(standalone ? runtimeIdentifiers : Enumerable.Empty<string>())
+                .WithRuntimeIdentifiers(runtimeIdentifiers)
                 .WithLockFile(LockFile)
                 .Build();
 
-            if (standalone && context.RuntimeIdentifier == null)
+            if (!context.IsPortable && context.RuntimeIdentifier == null)
```

Comment:
I don't think we need to go that far... negating a boolean is not exactly complicated :).